packages/taquito/src/constants.ts

```diff
@@ -2,7 +2,10 @@ export enum DEFAULT_GAS_LIMIT {
   DELEGATION = 10600,
   ORIGINATION = 10600,
   TRANSFER = 10600,
-  REVEAL = 1100,
+  REVEAL_TZ1 = 2000,
+  REVEAL_TZ2 = 2000,
+  REVEAL_TZ3 = 2000,
+  REVEAL_TZ4 = 2000,
```

Comment:
since we're passing 2000, does the node automatically default the return values to 2k in gas limit? or does it auto adjust and simulate accordingly?